Desktop/Regression_test_for_the_Desktop_Client.md

```diff
@@ -0,0 +1,288 @@
+# Regression test for the Desktop Client
+
+## Previous requirements
+
+Have a Desktop Client vX.X.X ready to be used with owncloud.
+
+Prepare a OC server with ssl actived and trusted certificates.
+
+Have a LDAP server ready to be used with owncloud.
+
+Have an Active Directory server ready to be used with owncloud.
+
+Have ready two external storage of your choice (e.g S3)
+
+## Testing
+
+### 1. Settings windows - Login
+
+TestID | Test Case | Steps to reprouce |Expected Result |Result| Related Comment
+------------ | ------------- | ------------- | ------- | ----- | ------
+1 | Update Installation | 1. You need to have installed a previous version 2. Update the new version ||:construction:| |
```

Comment:
I'd add some requirements to the previous version, such as to have: at least 2 accounts logged in, some folders selected to be synced and ask for confirmation before downloading x MB so that we may check that previous options are kept